include/deal.II/grid/reference_cell.h

```diff
@@ -201,7 +201,7 @@ class ReferenceCell
    * Return a Gauss-type quadrature matching the given reference cell (QGauss,
    * QGaussSimplex, QGaussPyramid, QGaussWedge).
    *
-   * @param[in] n_points_1D The number of quadrature points in each direction
+   * @param[in] n_points_1d The number of quadrature points in each direction
```

Comment:
You will then also need to replace it in the signature.

Reply:
Ah yes, good point. So fixed.